foundation/foundation-configuration-store/src/test/resources/configurations/example-env/aws-credentials.properties

```diff
@@ -0,0 +1,11 @@
+###
+# #%L
+# aiSSEMBLE::Foundation::Configuration::Store
+# %%
+# Copyright (C) 2021 Booz Allen
+# %%
+# This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+# #L%
+###
```

Comment:
S: probably worth a comment specifically noting that this is a test-only value for when it inevitably gets flagged by a secrets scanner.  Just helps folks maintaining or looking have a concise understanding (granted, it should be pretty contextual already since this is in the test folder...)